BUILDING.md

```diff
@@ -74,6 +74,7 @@ Feature constants are defined in [Common.props](src/NUnitFramework/Common.props)
  - `PARALLEL` enables running tests in parallel
  - `PLATFORM_DETECTION` enables platform detection
  - `THREAD_ABORT` enables timeouts and forcible cancellation
+ - `COM_APARTMENT` enables control of the thread apartment state
```

Comment:
Minor nit, but would something like `APARTMENT_STATE` or `THREAD_APARTMENT` be clearer? Many new developers probably don't know what COM is 😋

Reply:
That's fine. I changed it from `APARTMENT_STATE` just to be didactic about the naming and the shorter constant grew on me but I'll switch back for clarity.